embedded/store/indexer.go

```diff
@@ -163,6 +170,7 @@ func (idx *indexer) Close() error {
 
 	idx.stop()
 	idx.wHub.Close()
+	idx.store.releaseAllocTx(idx.tx)
```

Comment:
Since releasing of the tx slot has now moved to the Close method, there will be a tx leak if the Close method is not called correctly. There's one case where I'm aware of missing closed slot: https://github.com/codenotary/immudb/issues/873 - is it going to be any issue ?

Reply:
I don't think it will present an issue in this point, but cited issue needs to be solved.
The indexer is closed only when the store is closed.